bluemira/power_cycle/net/loads.py

```diff
@@ -394,25 +289,21 @@ def curve(self, time):
         curve: list[float]
             List of power values. [W]
         """
-        time = self._validate_curve_input(time)
-        preallocated_curve = np.zeros(len(time))
-
-        zip_of_sets = zip(self.loaddata_set, self.loadmodel_set)
-        for loaddata, loadmodel in zip_of_sets:
-            single_curve = self._single_curve(loaddata, loadmodel, time)
-            preallocated_curve += np.array(single_curve)
-        curve = preallocated_curve.tolist()
+        time = np.array(validate_list(time))
+        curve = np.zeros_like(time)
 
+        for loaddata, loadmodel in zip(self.loaddata_set, self.loadmodel_set):
+            curve += self._single_curve(loaddata, loadmodel, time)
         return curve
 
-    def _normalize_time(self, new_end_time):
+    def _normalise_time(self, new_end_time):
```

Comment:
I cannot stand "normalize" with an S! Let's go back to "Z"s everywhere! Hehehe...